runelite-client/src/main/java/net/runelite/client/plugins/xpglobes/XpGlobesOverlay.java

```diff
@@ -287,4 +296,13 @@ private void drawTooltipIfMouseover(Graphics2D graphics, XpGlobe mouseOverSkill,
 
 		xpTooltip.render(graphics);
 	}
+
+	private String getActionKillsText(Skill skillName)
+	{
+
+		return (combat.contains(skillName) && client.getLocalPlayer().getInteracting() != null)
```

Comment:
Also might just be me but I think using large ternary statements like this one only adds clutter and it doesn't save that much space from just using an if else block.

Reply:
I like that idea, I'll take a look at the idle notifier for combat and rework to be more similar to that. One thing though is that during testing it stayed on kills remaining even when switching styles (and so skills), so I think it might only be refreshed when that particular skill gains xp. As for timeout, is there a standard amount of time? or something like 30 seconds would be fine?